README.md

```diff
@@ -2,6 +2,18 @@
 CS325 Implementation assignment 3: Linear programming
 
 ## Setup
+
+To run python in your computer you can use jupyter notebook which can be done from command promt.
+
+For windows
+
+pip install jupyter notebook
```

Comment:
Could you go in to detail about this? Python3 can just be installed on Windows, and use the terminal. This is titled "For windows" but is not required to run on windows.